theme/static_src/src/djangonaut-space.css

```diff
@@ -45,8 +45,16 @@
 @layer components {
     .blog-container {
         background-color: #f8f8f8;
-        padding: 20px;
+        padding: 20px 40px;
         margin-bottom: 20px;
+
+        p {
+            color: var(--dark);
+            margin: 0 0 1em 0;
+        }
+        a {
+            color: var(--ds-purple) !important;
+        }
```

Comment:
These changes didn't really work 😞 
@sabderemane I want to have the following changes in the blog:
- links be purple
- text be dark
- paragraphs have margin underneath

Do you know where I need to put this?

Reply:
Do we need to put this as a class on all the `a` elements?